scripts/2.mutation-classifier.py

```diff
@@ -326,19 +326,45 @@ def get_threshold_metrics(y_true, y_pred):
 pd.crosstab(coef_df.feature_set, np.sign(coef_df.weight).rename('coefficient_sign'))
 
 
+# ### Top coefficients for covariates model
+
 # In[21]:
 
-# Top standardized coefficients
-(coef_df
-    .query("feature_set == 'full'")
-    .head(10)
-)
+coef_df.query("feature_set == 'covariates'").head(10)
 
 
-# ## Investigate the predictions
+# ### Top coefficients for full model
 
 # In[22]:
 
+coef_df.query("feature_set == 'full'").head(10)
```

Comment:
These updates look good to me. The notebook looks good with how it shows both the covariate loadings, the PCA loadings, and then the weights to the genes.